build_metal.sh

```diff
@@ -155,10 +168,18 @@ echo "INFO: Build tests: $build_tests"
 
 # Prepare cmake arguments
 cmake_args+=("-B" "$build_dir")
-cmake_args+=("-G" "Ninja")
```

Comment:
Why remove Ninja?

Reply:
This is an artifact of my previous attempt to do a riscv build back in April. Ninja had some issues. I just added that line back into the script and everything is working find. Thanks for finding this one and asking.

Reply:
I've corrected this issue; added ninja back into the script. the PR should be synchronized now.